requirements-dev.txt

```diff
@@ -0,0 +1,142 @@
+anyio==4.3.0
+appnope==0.1.4
+argon2-cffi==23.1.0
+argon2-cffi-bindings==21.2.0
+arrow==1.3.0
+asttokens==2.4.1
+async-lru==2.0.4
+attrs==23.2.0
+aws-cdk-lib==2.130.0
+aws-cdk.asset-awscli-v1==2.2.202
+aws-cdk.asset-kubectl-v20==2.1.2
+aws-cdk.asset-node-proxy-agent-v6==2.0.1
+aws-xray-sdk==2.13.0
+Babel==2.14.0
+beautifulsoup4==4.12.3
+bleach==6.1.0
+boto3==1.34.50
+botocore==1.34.50
+cachetools==5.3.3
+cattrs==23.2.3
+certifi==2024.2.2
+cffi==1.16.0
+charset-normalizer==3.3.2
+click==8.1.7
+comm==0.2.2
+constructs==10.3.0
+contourpy==1.2.0
+cryptography==41.0.7
+cycler==0.12.1
+debugpy==1.8.1
+decorator==5.1.1
+defusedxml==0.7.1
+dill==0.3.6
+dnslib==0.9.24
+dnspython==2.6.1
+ecdsa==0.18.0
+executing==2.0.1
+fastjsonschema==2.19.1
+fonttools==4.49.0
+fqdn==1.5.1
+h11==0.14.0
+httpcore==1.0.4
+httpx==0.27.0
+idna==3.6
+importlib_resources==6.1.2
+ipykernel==6.29.3
+ipython==8.22.2
+isoduration==20.11.0
+jedi==0.19.1
+Jinja2==3.1.3
+jmespath==1.0.1
+jsii==1.94.0
+json5==0.9.22
+jsonpointer==2.4
+jsonschema==4.21.1
+jsonschema-specifications==2023.12.1
+jupyter-events==0.9.1
+jupyter-lsp==2.2.4
+jupyter_client==8.6.1
```

Comment:
Are all these dependencies really needed?

Reply:
I have a sample consumer as jupyter notebook in a follow up pr to help load and visualize data from redshift